build/build_deploy.sh

```diff
@@ -8,13 +8,20 @@ CURRENT_DIR=$(dirname "$0")
 
 BASE_IMG="managed-cluster-validating-webhooks"
 QUAY_IMAGE="quay.io/app-sre/${BASE_IMG}"
-IMG="${BASE_IMG}:latest"
-
+IMG="${BASE_IMG}:${VERSION_MAJOR}.${VERSION_MINOR}-${GIT_HASH}"
+VERSION_MAJOR=0
+VERSION_MINOR=1
+SELECTOR_SYNC_SET_TEMPLATE_DIR=templates
+YAML_DIRECTORY=deploy
+SELECTOR_SYNC_SET_DESTINATION=templates/00-osd-managed-cluster-validating-webhooks.selectorsyncset.yaml.tmpl
+REPO_NAME=managed-cluster-validating-webhooks
 GIT_HASH=$(git rev-parse --short=7 HEAD)
 
 # build the image
 BUILD_CMD="docker build" IMG="$IMG" make docker-build
```

Comment:
I'm not getting this make docker-build at the end of the docker build command.

Is it supposed to be here? If that's the case we don't have a `docker-build` handle in this repo's Makefile